### PR TITLE
Feature/8

### PR DESCRIPTION
**Pull request by YumeAyasaki**
In this PR, I have:
- Add sass to the project...
- Create global styling, including variables, mixins (although there's nothing there), and some stylings for headings and text...
- Make a Test page to demo some of the changes...

Note: I think we should unify (?) more things, maybe size for phones and tablets (there're many phone sizes, I just go with the 390px width on Figma) and maybe make a mixin for that. And maybe some other mixins if other members want to use...
And what is PR, again...?

### CODE REVIEW

File: styles/globals.scss [hunk 1]
```diff
@@ -0,0 +1,68 @@
+@use "/styles/variables" as variables;
+@use "/styles/variables" as mixins;
+
+@import url('https://fonts.googleapis.com/css2?family=Lora'); // Font Lora
+
+// Note: Font weight - https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight
+// Note: Ask about bold and bolder
+
+h1 { // Lora 32px bold
```
Comment (PaulHo0501) [inline styles/globals.scss]: Change these styling into mixins or class. This will reduce the amount of repeated code.

**Review by PaulHo0501 — COMMENTED** [review]
Please check the comment. The purpose of a global stylesheet is not to style individual HTML elements but to make mixins and classes that can be imported and used everywhere, and also, to reduce the amount the code needed to achieve the styling

File: styles/globals.scss (hunk 1 above)
Comment (PaulHo0501) [inline styles/globals.scss]: Add another comment here to request changes.

**Review by PaulHo0501 — CHANGES_REQUESTED** [review]

**Comment by YumeAyasaki** [comment]
Update PR:
- Change global styling to class...
- Add a mixin for Lora font...
- Edit the Test page according to the new global styling...?

Note: I want to make a mixin for phone media query, do we use max-width or min-width for phone size...?

File: styles/globals.scss [hunk 2]
```diff
@@ -1,39 +1,29 @@
...
-  font-size: 28px;
+.sub-header { // Lora 28px bold
+  @include mixins.fastLoraFont(700, 28px);
 }
 
-h3 { // Lora 24px bold
-  font-family: "Lora";
-  font-weight: 700;
-  font-size: 24px;
+.sub-header-2 { // Lora 24px bold
```
Comment (PaulHo0501) [inline styles/globals.scss]: Please rename classes using camelUpperCase (eg. subHeader, subHeader2, ...)

**Review by PaulHo0501 — CHANGES_REQUESTED** [review]

**Comment by YumeAyasaki** [comment]
Update PR:
- Change name class...

File: yarn.lock [hunk 3]
```diff
@@ -1,1608 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (PaulHo0501) [inline yarn.lock]: Solve the conflict with develop branch

**Review by PaulHo0501 — CHANGES_REQUESTED** [review]

**Comment by PaulHo0501** [comment]
LGTM

**Pull request merged**